.github/workflows/release_validate.yml

```diff
@@ -1,101 +1,33 @@
 name: "Release (validate)"
 
 on:
-  workflow_call:
-    inputs:
-      version:
-        required: true
-        type: string
+  schedule:
+    - cron: 0 */6 * * *" # every six hours
 
 jobs:
   validate:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with:
-          submodules: recursive
-          fetch-depth: 0
 
-      - name: Set up Dotnet
-        uses: actions/setup-dotnet@v3
-        with:
-          dotnet-version: 6.0.x
+      - uses: actions/setup-dotnet@v3
+      - uses: actions/setup-go@v4
+      - uses: actions/setup-java@v3
+      - uses: actions/setup-node@v3
 
-      - name: Set up Java
-        uses: actions/setup-java@v3
-        with:
-          java-version: '11'
-          distribution: 'temurin'
+      - run: ./scripts/install_zig.sh
+      
+      - run: ./zig/zig build dotnet_client:ci -- --release-validate
+      - run: ./zig/zig build go_client:ci     -- --release-validate
+      - run: ./zig/zig build java_client:ci   -- --release-validate
+      - run: ./zig/zig build node_client:ci   -- --release-validate
```

Comment:
That's going to be completely broken for know, as we don't have a proper release with u128 balances. 


But yeah, the idea here is to run this on the clock, to test latest releases.

Reply:
> But yeah, the idea here is to run this on the clock, to test latest releases.

Do we not also want to run it after publish, to catch any issues quicker?

Reply:
Yes, though what gives me pause is "eventual consistency" of publishing process --- my understnading is that some package registiries don't guarantee that a new version becomes available immediately after publishing. But we probably should do something along the lines of "run 10 minutes after the publish".